contents.json

```diff
@@ -13076,6 +13076,16 @@
       "suggested_by": "@dkhamsing",
       "stars": 36
     },
+    {
+      "title": "Awesome ML",
+      "category-ids": ["developer"],
+      "description": "Discover, download, compile & launch different image processing CoreML models on iOS.",
```

Comment:
Could we use this: "Discover, download, compile & launch different image processing CoreML models" instead?